app/src/main/java/zegreanu/cristi/geoquiz/CheatActivity.kt

```diff
@@ -27,11 +34,17 @@ class CheatActivity : AppCompatActivity() {
                 answerTextView.setText(R.string.true_button)
             else
                 answerTextView.setText(R.string.false_button)
-            setAnswerShownResult(true)
+            isAnswerShown = true
+            setAnswerShownResult()
         }
     }
 
-    private fun setAnswerShownResult(isAnswerShown: Boolean) {
+    override fun onSaveInstanceState(outState: Bundle) {
+        super.onSaveInstanceState(outState)
+        outState.putBoolean(KEY_ANSWER_SHOWN, isAnswerShown)
```

Comment:
New line after calling super. Hmm - is there a way to include this rule in detekt or ktlint?

Reply:
I will try to include it with next chapter